debian/control

```diff
@@ -20,5 +20,8 @@ Depends: php-mysql,
          php (>= 7~),
          libapache2-mod-php,
          ${misc:Depends}
-Description: web-accessible database for longitudinal multi-site studies
- Whis should I install this one over XNAT?
+Description: LORIS is a web-based data management software. 
```

Comment:
Please no trailing period
This short description is already an ending for "loris is a" sentence